Apromore-Core-Components/Apromore-Manager/src/test/java/org/apromore/test/service/impl/ProcessServiceImplUnitTest.java

```diff
@@ -919,6 +919,32 @@ public void testDeleteProcessModel_BranchHasOnlyOnePMV() throws Exception {
   }
 
 
+  /**
+   * Test the {@link ProcessServiceImpl#sanitizeBPMN} method.
+   */
+  @Test
+  public void testSanitizeBPMN() throws Exception {
+
+      // Check that nothing changes for an innocuous BPMN file
+      Assert.assertEquals(
+          CharStreams.toString(new InputStreamReader(getResourceAsStream("BPMN_models/Cyclic.bpmn"))),
+          CharStreams.toString(new InputStreamReader(ProcessServiceImpl.sanitizeBPMN(getResourceAsStream("BPMN_models/Cyclic.bpmn")))) + "\n"
```

Comment:
The XSL transformation elides the trailing newline, which is why I have to manually append it here.  It'd be helpful for someone on Windows to confirm the XML processing isn't sensitive to the system line ending.